modules/utility/init.zsh

```diff
@@ -12,6 +12,9 @@ if (( terminfo[colors] >= 8 )); then
   # GNU colours are used by completion module for all OSTYPEs
   (( ! ${+LS_COLORS} )) && export LS_COLORS='di=1;34:ln=35:so=32:pi=33:ex=31:bd=1;36:cd=1;33:su=30;41:sg=30;46:tw=30;42:ow=30;43'
 
+  # Set completion colors to LS_COLORS
+  zstyle ':completion:*' list-colors $LS_COLORS
```

Comment:
This is already in the [completion module](https://github.com/zimfw/zimfw/blob/56b61e211d2e6abed919d31845fcb89fc6ba15fe/modules/completion/init.zsh#L62).

Reply:
The reason for the move is to allow the `completion` module to be loaded at an arbitrary location and stop `compdef` errors.

Reply:
I like the idea of making modules independent from each other.

Which `compdef` errors are you referring to?

Reply:
For example, the `osx` module that I have installed from `Oh My Zsh`, runs `compdef _man man-preview`.

Reply:
That should be fixed by sourcing the `osx` module **after** the `completion` one. As `completion` runs compinit, which defines the compdef function.

Reply:
@ericbn I know. But I still think we should make the completion module independent.